lib/ansible/config/base.yml

```diff
@@ -1709,19 +1709,6 @@ INVENTORY_UNPARSED_IS_FAILED:
   ini:
   - {key: unparsed_is_failed, section: inventory}
   type: bool
-JINJA2_NATIVE_WARNING:
-   name: Running older than required Jinja version for jinja2_native warning
-   default: True
-   description: Toggle to control showing warnings related to running a Jinja version
-                older than required for jinja2_native
-   env:
-    - name: ANSIBLE_JINJA2_NATIVE_WARNING
-      deprecated:
```

Comment:
this only deprecated the env var, the whole entry should have had a deprecation notice, those using ini, for example would not get notice.

Reply:
we probalby need to 'redeprecate', but the entire entry now.